src/components/AdministrationPage/AdministrationPage.tsx

```diff
@@ -158,32 +193,107 @@ export const AdministrationPage = () => {
                           }}
+                          onBlur={() => {
+                            setAnnouncementTouched(true);
+                          }}
                         />
                       </FormGroup>
                     </GridItem>
+                    {isMaintenanceModeOn && (
+                      <GridItem span={12}>
+                        <FormGroup
+                          label="* ETA Time"
+                          fieldId="eta-time"
+                          helperTextInvalid={invalidEtaTime() ? 'Required field.' : null}
+                          helperTextInvalidIcon={<ExclamationCircleIcon />}
+                          validated={'error'}
+                        >
+                          <DatePicker
+                            required
+                            isDisabled={!isMaintenanceModeOn || isEtaNa}
+                            name="form-etaTime"
+                            id="eta-time"
+                            placeholder="yyyy-MM-dd hh:mm (UTC)"
+                            dateFormat={transDateFormat}
+                            onClick={() => {
+                              setEtaTouched(true);
+                            }}
+                            dateParse={(dateString) => {
+                              return new Date(dateString);
+                            }}
+                            value={etaTime}
+                            onBlur={(value: string) => {
+                              setEtaTime(value);
+                            }}
+                            onChange={(value: string) => {
+                              setEtaTime(value);
+                            }}
+                            aria-invalid={invalidEtaTime()}
+                          />
+                          &nbsp;&nbsp;
+                          <Switch
+                            id="eta-na-switch"
+                            label=" N/A:"
+                            labelOff=" N/A:"
+                            hasCheckIcon
+                            isChecked={isEtaNa}
+                            onChange={() => {
+                              setIsEtaNa(!isEtaNa);
+                              setEtaTouched(true);
+                            }}
+                            isReversed
+                          />
+                        </FormGroup>
+                      </GridItem>
+                    )}
+                    {isMaintenanceModeOn && (
+                      <GridItem span={12}>
+                        <FormGroup label="Example" fieldId="eta-time">
+                          {isMaintenanceModeOn && (
+                            <TopBarInfo>
+                              Maintenance Mode - PNC system is in the maintenance mode, no new build requests are accepted.
+                              Reason: {announcementMessage ? announcementMessage : 'N/A'}, ETA: {etaTime ? etaTime : 'N/A'}
+                            </TopBarInfo>
```

Comment:
There is a problem with this toolbar:
When you close the example toolbar and move to another page, for example Projects, also top bar on top side of page will disappear.
Solution:
Add parameter to the TopBar component which would make close button disabled (with default value enabled).